app/controllers/slides_controller.rb

```diff
@@ -97,11 +97,11 @@ def update
     @slide.assign_attributes(slide_params)
     if @slide.update_attributes(slide_params)
       if slide_convert_status == 0
-        send_message({ id: @slide.id, key: @slide.key }.to_json)
+        CloudConfig::SERVICE.send_message({ id: @slide.id, key: @slide.key }.to_json)
       end
       redirect_to slide_path(@slide.id)
     else
-      render :edit
+      render "slides/#{CloudConfig::service_name}/edit"
```

Comment:
__[rubocop]__
* Style/ColonMethodCall: Do not use `::` for method calls. [:ref](https://github.com/bbatsov/ruby-style-guide#double-colons)